File: src/mypack/Item.java
package mypack;

/**
 * Created by xBons_000 on 06-08-2014.
 */
public class Item {
    public String name;
    public String description;
    public double price;
    public int id;
    public int quantity;

    public Item(String name, String description, double price, int id, int quantity) {
        this.name = name;
        this.description = description;
        this.price = price;
        this.id = id;
        this.quantity = quantity;
    }

    public void add(int quantity) {
        this.quantity += quantity;
    }

    public void substract(int quanity) {
        this.quantity -= quantity;
    }
}

File: web/jslibs/Item.js
/**
 * Created by xBons_000 on 13-08-2014.
 */
function Item(){
    this.name = "";
    this.description = "";
    this.price = 0.0;
    this.id = null;
    this.quantity = 0;

    // RECEIVES A PRODUCT WITH THE FORMAT.
    this.setFields = function (e) {
        //if it comes frm db.
        if(e.hasOwnProperty("na_product")) {
            this.name = e["na_product"];
            this.description = e["de_product"];
            this.price = e["pr_product"];
            this.id = e["id_product"];
            this.quantity = e["qu_product"];
        //If it comes as an item.
        } else if(e.hasOwnProperty("name")){
            this.name = e.name;
            this.description = e.description;
            this.price = e.price;
            this.id = e.id;
            this.quantity = e.quantity;
        } else {
            console.log("Format not valid");
        }
    };

    this.add = function (q) {
        this.quantity += q;
    };

    this.substract = function (q) {
        this.quantity -= q;
    };
}